composer.json

```
@@ -26,12 +26,13 @@
         }
     },
     "require": {
-        "php": "^7.0",
+        "php": "^7.0 || ~8.1",
         "symfony/config": "^2.8 || ^3.0 || ^4.0",
         "symfony/console": "^2.8 || ^3.0 || ^4.0",
+        "symfony/http-client": "^5.4",
         "symfony/http-kernel": "^2.8 || ^3.0 || ^4.0",
         "symfony/dependency-injection": "^2.8,>=2.8.51 || ^3.0,>=3.4.26 || ^4.1,>=4.1.12 || ^4.2,>=4.2.7",
-        "symfony/process": "^2.8 || ^3.0 || ^4.0",
+        "symfony/process": "^2.8 || ^3.0 || ^4.0 || ^5.4",
         "symfony/options-resolver": "^2.8 || ^3.0 || ^4.0",
```

```suggestion
        "symfony/options-resolver": "^2.8 || ^3.0 || ^4.0 || ^5.0",
```

@Rheed I believe we have to allow v5 of the `symfony/options-resolver` because I can not upgrade `symfony/form` because of that:

https://github.com/symfony/form/blob/e96eea999cf185960b179e96f04b9fa2ac2195d0/composer.json#L22

and `php composer.phar why-not symfony/options-resolver ^5.0` gives me this:

```
prgtw/healthchecks-bundle 0.9.0 requires symfony/options-resolver (^2.8 || ^3.0 || ^4.0)
```

---

or maybe it isn't your goal to add symfony v5 support as well in the single PR?

---

> Add php8.1 compatibility

So, my guess it's not about the Symfony

---

To be honest, I'm confused what more is necessary without just widening `php` dependency in `composer.json`